Automattic/jetpack
# Extensions: update usage of isCurrentUserConnected from package

#### Changes proposed in this Pull Request:

Now that we've extracted this function into our shared package (see #27923), let's import it from there, and get rid of the Jetpack plugin implementation.

#### Other information:


#### Jetpack product discussion

* 1386-gh-dotcom-forge

#### Does this pull request change what data or activity we track or use?

* No


#### Testing instructions:

* In the Jetpack plugin, while connected to WordPress.com, check blocks like the Instagram gallery and see that they still work.
